core/domain/config_domain.py

```diff
@@ -172,6 +182,14 @@ def set_value(self, committer_id, raw_value):
             model_instance.id: model_instance.value})
 
     def normalize(self, value):
+        """Validate the given object using the schema, normalizing if necessary.
```

Comment:
Validates the given object using the schema and normalizes if necessary.

Reply:
Done, Thanks!